ascii.py

```diff
@@ -5,65 +5,62 @@
 from math import floor
 from PIL import Image
 
+TERMINAL_DIMENSIONS = [180, 41, 10, 22]
```

Comment:
constants should ideally be outside all other code

Reply:
yup